external-api/src/v1/controllers/geospatial-addresses.controller.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
```

Comment:
Please remove.

Reply:
It will take quite some time. 

Maybe it can be as part of other `Typescript improvement` PR.